helm/external-dns-app/templates/serviceaccount/configmap.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if not .Values.serviceAccount.create }}
```

Comment:
this is breaking the current behavior, where setting `.Values.serviceAccount.create` to false is meant to use an existing SA.

Reply:
I've introduced `.Values.serviceAccount.useIRSA` to do this instead. This does mean that these two values are incompatible (create=false and useIRSA=true), but I've pointed it out in the values.